gitpod-io/website
# [Self-hosted] Add Backup and Restore + Disaster Recovery Documentation

## Description
Adds two pages to the docs:
- How to backup and restore guide (relates to https://github.com/gitpod-io/gitpod/issues/10515)
- Background on Disaster recovery vs. Gitpod

--> Created a single PR for both because they reference each other. 

## Related Issue(s)

Fixes #
- https://github.com/gitpod-io/gitpod/issues/10515
- https://github.com/gitpod-io/gitpod/issues/10516

## How to test
Open netlify preview and navigate to docs>self-hosted and:
- Operational guides>Backup 
  1. Go through Guide
  2. Create self-hosted instance
  3. ❗important ❗Run through the steps to make sure you can do a backup and restore as described in the steps.
- Background>Disaster Recovery
  1.  Go through Guide

## Release Notes

```release-note
Added documentation around backing-up and restoring Self-Hosted Gitpod as well as on Disaster Recovery
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2387"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2387"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
